spec/dynamoid_advanced_where/search_tests/start_key_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'spec_helper'
+
+RSpec.describe "Start key results" do
+  let(:large_string) { "a" * 400000 }
```

Comment:
is this to make it hit that 1MB limit so it doesn't return everything?

Reply:
Yes

Reply:
also DynamoDB items are limited to 400Kb, so this is as big as I could make them